docs/docs/view-config-json.md

```diff
@@ -0,0 +1,171 @@
+|-------|-----|
+| `1.0.0`| Support for the coordination space was added in this version. We recommend that you use this version. |
+
+### `initStrategy`
+- Type: `string`
+
+The coordination space initialization strategy. The initialization strategy determines how missing coordination objects and coordination scope mappings are initially filled in.
+
+|Value|Notes|
+|-------|-----|
+| `auto`| Recommended. This strategy will allow you to omit some or all coordination scope mappings. |
+| `none`| Use this strategy if you would like to define all coordination scope mappings yourself. |
+
+:::note
+If you find yourself following certain patterns when initializing coordination spaces manually, please let us know so that we may work with you to implement this pattern as a supported strategy.
+:::
+
+### `datasets`
+- Type: `object[]`
+
+The datasets array stores a list of dataset objects. Each dataset object must contain a unique ID `uid`, a list of file objects `files`, and an optional `name`.
+
+#### `uid`
+- Type: `string`
+
+A unique ID for the dataset. Required.
+
+#### `name`
+- Type: `string`
+
+A human-readable name for the dataset. Optional.
+
+#### `files`
+- Type: `object[]`
+
+The files array stores a list of file objects for a dataset. Each dataset may have one file of each `type`. File objects must contain a `type` ("data type"), `fileType` ("file type"), and a `url`. For more information about data types and file types, please visit our [Supported File Types](/docs/data-file-types/index.html) documentation page. We don't associate any semantics with URL strings.
+
+```json
+...,
+"datasets": [
+    {
+        "uid": "my-dataset",
+        "name": "My amazing dataset",
+        "files": [
+            {
+                "type": "cells",
+                "fileType": "cells.json",
+                "url": "http://example.com/a.json"
+            },
+            {
+                "type": "cell-sets",
+                "fileType": "cell-sets.json",
+                "url": "http://example.com/b.json"
+            }
+        ]
+    }
+],
+...
+```
+
+### `layout`
+- Type: `object[]`
+
+The layout property defines which visualization (and controller) components will be rendered, how they will be arranged on the screen, and optionally how they will map onto coordination scopes. Each layout object represents one "component" or "view", and must contain a component name `component`, width `w` and height `h`, and horizontal position `x` and vertical position `y`. Components are arranged in a grid with 12 columns and a dynamic number of rows. Optionally, each component may contain the properties `coordinationScopes` and `props`.
+
+For more information about the components that are available, please visit the [Visualization Components](/docs/components/index.html) documentation page.
+
+For more information about the coordination types that are available, please visit the [Coordination Types](/docs/coordination-types/index.html) documentation page.
```

Comment:
Yes, I have updated the auto-generated coordination type listing to include which component uses each. As for why the lookup exists, I think using a subset may help zustand to avoid unnecessary re-renders